andrea/js/app.js

```diff
@@ -0,0 +1,67 @@
+// buttons //
+var signUp = document.querySelector(".navbar-sign");
+var logIn = document.querySelector(".navbar-login");
+
+var create_btn = document.querySelector(".create-btn");
+
+// form //
+var form = document.querySelector(".register-form");
+
+// passwords //
+var password = document.getElementById("password");
+var confirm_password = document.getElementById("confirm-password");
+
+// show register form //
+function showRegister() {
+    document.querySelector(".title").style.display = "none";
+    document.querySelector(".description").style.display = "none";
+    signUp.style.display = "none";
+    logIn.style.display = "none";
+    form.classList.add("form__show");
+    document.querySelector(".register-title").style.display = "flex";
```

Comment:
Assign the return value of this selector to a variable at the top, lets try to keep all queries at the top so whoever reads this code doesnt need to jump around to find which are all the elements that we query for in our app, it makes the code more readable